### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by dictoon**

### CODE REVIEW

File: src/appleseed/renderer/modeling/bsdf/disneybrdf.cpp [hunk 1]
```diff
@@ -338,12 +338,12 @@ namespace
             m_inputs.declare("base_color", InputFormatSpectralReflectance);
             m_inputs.declare("subsurface", InputFormatFloat, "0.0");
             m_inputs.declare("metallic", InputFormatFloat, "0.0");
-            m_inputs.declare("specular", InputFormatFloat, "0.5");
+            m_inputs.declare("specular", InputFormatFloat, "0.0");
             m_inputs.declare("specular_tint", InputFormatFloat, "0.0");
             m_inputs.declare("anisotropic", InputFormatFloat, "0.0");
             m_inputs.declare("roughness", InputFormatFloat, "0.5");
```
Comment (est77) [inline src/appleseed/renderer/modeling/bsdf/disneybrdf.cpp]: Could we change this one too, now that we are modifying the defaults? I always thought that 0.5 is too much.

Comment (dictoon) [inline src/appleseed/renderer/modeling/bsdf/disneybrdf.cpp]: You mean roughness? We could set it to 0.0 by default.

Comment (est77) [inline src/appleseed/renderer/modeling/bsdf/disneybrdf.cpp]: Yes. Maybe something like 0.1?

Comment (dictoon) [inline src/appleseed/renderer/modeling/bsdf/disneybrdf.cpp]: Ok.

**Pull request merged**